kuma/users/tests/test_views.py

```diff
@@ -127,12 +127,15 @@ def tearDown(self):
     def _get_current_form_field_values(self, doc):
         # Scrape out the existing significant form field values.
         form = dict()
-        for fn in ('username', 'email', 'fullname', 'title', 'organization',
-                   'location', 'irc_nickname', 'bio', 'interests'):
-            form[fn] = doc.find('#user-edit *[name="user-%s"]' %
-                                fn).val()
-        form['country'] = 'us'
-        form['format'] = 'html'
+        prefix = 'user-'
+        fields = (
+            'username', 'email', 'fullname', 'title', 'organization',
+            'location', 'irc_nickname', 'bio', 'interests')
+        for field in fields:
+            value = doc.find('#user-edit *[name="user-%s"]' % field).val()
```

Comment:
Though we could go on forever with using the prefix var instead of hard-coding. Meh. Up to you if it bothers you enough ...